### PR TITLE
Remove redundant yum package installation

### PR DESCRIPTION
**Pull request by abhay-krishna**
Removing yum installs of packages that are already on the builder-base image.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/145#" title="Author self-approved">abhay-krishna</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [abhay-krishna]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: projects/kubernetes-sigs/image-builder/buildspecs/build-1-21-ubuntu-ova.yml
Comment (jaxesn) [inline projects/kubernetes-sigs/image-builder/buildspecs/build-1-21-ubuntu-ova.yml:21]: You want to just put this on the builder base?

**Review by jaxesn — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/145#" title="Author self-approved">abhay-krishna</a>*, *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/145#pullrequestreview-799399770" title="LGTM">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [abhay-krishna,jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: projects/kubernetes-sigs/image-builder/buildspecs/build-1-21-ubuntu-ova.yml
Comment (abhay-krishna) [reply projects/kubernetes-sigs/image-builder/buildspecs/build-1-21-ubuntu-ova.yml:21]: I did so [here](https://github.com/aws/eks-distro-build-tooling/commit/7a4552d57c9eb4bc35428ecf537bed15fb135c83) the other day, but the ensuing postsubmit failed because of the ARM build issues. Will remove these after a successful builder-base build.

**Pull request merged**
